inference-engine/tests/functional/inference_engine/ngraph_reader/interpolate_tests.cpp

```diff
@@ -220,3 +220,183 @@ TEST_F(NGraphReaderTests, ReadInterpolate2Network) {
+        <layer id="2" name="scales" type="Const" precision="FP32" version="opset1">
+            <data offset="8" size="8"/>
+            <output>
+                <port id="1" precision="FP32">
+                    <dim>2</dim>
+                </port>
+            </output>
+        </layer>
+        <layer id="3" name="axes" type="Const" precision="I32" version="opset1">
+            <data offset="16" size="8"/>
+            <output>
+                <port id="1" precision="I32">
+                    <dim>2</dim>
+                </port>
+            </output>
+        </layer>
+        <layer id="4" name="interpolate" type="Interpolate" precision="FP32" version="opset4">
+            <data antialias="0" coordinate_transformation_mode="asymmetric" cube_coeff="123" mode="nearest" nearest_mode="floor" pads_begin="2,3,4,5" pads_end="6,7,8,9" shape_calculation_mode="sizes"/>
+            <input>
+                <port id="0">
+                    <dim>1</dim>
+                    <dim>2</dim>
+                    <dim>300</dim>
+                    <dim>300</dim>
+                </port>
+                <port id="1">
+                    <dim>2</dim>
+                </port>
+                <port id="2">
+                    <dim>2</dim>
+                </port>
+                <port id="3">
+                    <dim>2</dim>
+                </port>
+            </input>
+            <output>
+                <port id="4" precision="FP32">
+                    <dim>9</dim>
+                    <dim>12</dim>
+                    <dim>600</dim>
+                    <dim>900</dim>
+                </port>
+            </output>
+        </layer>
+        <layer name="output" type="Result" id="5" precision="FP32" version="opset1">
+            <input>
+                <port id="0">
+                    <dim>1</dim>
+                    <dim>2</dim>
+                    <dim>50</dim>
+                    <dim>60</dim>
+                </port>
+            </input>
+        </layer>
+    </layers>
+    <edges>
+        <edge from-layer="0" from-port="0" to-layer="4" to-port="0"/>
+        <edge from-layer="1" from-port="1" to-layer="4" to-port="1"/>
+        <edge from-layer="2" from-port="1" to-layer="4" to-port="2"/>
+        <edge from-layer="3" from-port="1" to-layer="4" to-port="3"/>
+        <edge from-layer="4" from-port="4" to-layer="5" to-port="0"/>
+    </edges>
+</net>
+)V0G0N";
+    std::string modelV5 = R"V0G0N(
+<net name="Network" version="5" precision="FP32" batch="1">
+    <layers>
+        <layer id="0" name="data" precision="FP32" type="Input">
+            <output>
+                <port id="0">
+                    <dim>1</dim>
+                    <dim>2</dim>
+                    <dim>300</dim>
+                    <dim>300</dim>
+                </port>
+            </output>
+        </layer>
+        <layer id="1" name="output_shape" precision="I32" type="Const">
+            <output>
+                <port id="0">
```

Comment:
Maybe I64? Infer function of Interpolate-1 or Interpolate-4 reads this input as a constant with `int64_t` values?

Reply:
it does not really matter. This input is *converted* to `int64_t` type using `cast_vector` function. I chose I32 type here because it was simpler to write a test in this case. Because after conversion to CNNNetwork this constant will be converted to I32 and I need to have different weights for the reference model which is painful.